eclipse-theia/theia
# output: fix `runtime-import-check` errors

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/10119
Fixes: https://github.com/eclipse-theia/theia/issues/4306

The pull-request updates **output** to resolve `runtime-import-check` errors where _common_ was incorrectly importing code from _browser_ which caused downstream issues due to incompatible runtimes (_node_ ends up pulling _browser_ code through _common_).

#### How to test


- the application should successfully build and start
- confirm that the output-view can be opened, the sample channel should display

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
